server/@types/viewModels/index.d.ts

```diff
@@ -212,8 +212,4 @@ declare module 'viewModels' {
     pageUrls: Array<string>
     currentPageIndex: number
   }
-
-  export interface UpdateInductionQuestionSet {
```

Comment:
This was essentially the flag that we held on the session that would be used in the controller code to determine if we were in the middle of a "change question set" type journey